### PR TITLE
STR-146 -- FE: Implement Information Slides #19, #21, #22, #23, #24 into Alcohol Module

### PR DESCRIPTION
**Pull request by frlzjosh**
# Description
'Write a meaningful description of your task'
 - I need to implement the informational slides #19, #21, #22, #23 and #24 of the alcohol module.
- Create functionality that will allow us to iterate through these slide numbers
# Testing
'What does the reviewer need to do to test your PR?'
1. Login as `april.feldman`
2. click on any module
3. you will then be redirected to the informational slides
4. there is functionality that doesn't allow the user to click on continue until after 5 seconds of the page
  - make sure that works
  - it would help to look simultaneously look at vuex in devtools
5. you can click back on continue
  - when you click "back" you're able to click on "continue"
6. Review the json file to see that it works and is a good template for the future objects to come
- run `yarn jest-unit`
# Installation
'Are there any dependencies the reviewer needs to download?'
- you might need to run `yarn install` incase you haven't used `jest-extended`

### CODE REVIEW

File: resources/js/store/modules/slides/state.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default {
   slides: {
```
Comment (ekno27) [inline resources/js/store/modules/slides/state.js]: change name to something that indicates the object a little bit better, something along the lines of `slideContent` or somethin

File: tests/jsUnit/components/modules/template.spec.js [hunk 2]
```diff
@@ -1,9 +1,69 @@
 import slideStore from './../../../../resources/js/store/modules/slides/'
+import 'jest-extended'
+const state = {
+  slides: {},
```
Comment (ekno27) [inline tests/jsUnit/components/modules/template.spec.js]: same applies here

File: resources/js/components/modules/moduleFooter.vue [hunk 3]
```diff
@@ -49,10 +54,24 @@ export default {
...
+          isAbleToProceed: true,
+          slide_index: this.current_slide -= 1
+        }
+        this.allowUserToContinue(payload)
+      }
+    },
     proceedAndHideContent: function () {
-      this.allowUserToContinue(false)
-      this.current_slide += 1
+      // if(this.current_slide < Object.keys(this.amountOfSlides).length + 1){
```
Comment (ekno27) [inline resources/js/components/modules/moduleFooter.vue]: remove comment :0

**Review by ekno27 — CHANGES_REQUESTED** [review]
few things: 
1. When dashboard is clicked and module is accessed, no info is displayed and theres console errors 
2.  continue and back buttons are inconsistent.  For example, If i press back before pressing continue first, for example, I am unable to go back or forward
3. Only time back button works is when you reach the end, but if you try to go forward or back it won't let you

File: resources/js/store/modules/slides/state.js (hunk 1 above)
Comment (frlzjosh) [reply resources/js/store/modules/slides/state.js]: iiiight how's `importedJSONSlides` ?

File: tests/jsUnit/components/modules/template.spec.js (hunk 2 above)
Comment (frlzjosh) [reply tests/jsUnit/components/modules/template.spec.js]: gotchu

File: resources/js/components/modules/moduleFooter.vue (hunk 3 above)
Comment (frlzjosh) [reply resources/js/components/modules/moduleFooter.vue]: fffffsho

**Review by ekno27 — APPROVED** [review]

**Review by andrewMETALAB — APPROVED** [review]
GREAT WORK 👏

**Pull request merged**